src/routine.js

```diff
@@ -1,8 +1,10 @@
 const client = require('./client')
 
+
 const { ASANA_PROJECT = '1152701043959235' } = process.env
 
-const pastFilter = task => new Date(`${task.due_at ? task.due_at : `${task.due_on}T23:59:59.999Z`}`) < new Date()
+const pastFilter = (task) => new Date(`${task.due_at ? task.due_at : `${task.due_on}T23:59:59}`}`)
```

Comment:
i think there was a reason to do `.999Z` to make it fully compliant with timestamp parsing.